config/routes.rb

```diff
@@ -4,5 +4,14 @@
     resources :users
   end
   devise_for :users
+
+  resources :projects do
+    resources :issues
```

Comment:
add nested routes for comments over here

Reply:
I only need the resource in create action and in here the resource in issue. I have no use for project in create action so the link integrity could not be verified for nested resource. It would lead to routes like 'projects/1/issues/1/comments/1' and 'projects/2/issues/1/comments/1' to be possible and lead to the same action.